test/optimise.jl

```diff
@@ -1,22 +1,39 @@
-using IntervalArithmetic, IntervalOptimisation
+﻿using IntervalArithmetic, IntervalOptimisation
 using Test
 
 @testset "IntervalOptimisation tests" begin
 
-    @testset "Minimise in 1D" begin
+    @testset "Minimise in 1D using default data structure i.e HeapedVector" begin
         global_min, minimisers = minimise(x->x, -10..10)
         @test global_min ⊆ -10 .. -9.999
         @test length(minimisers) == 1
         @test minimisers[1] ⊆ -10 .. -9.999
 
-        global_min, minimisers = minimise(x->x^2, -10..11, 1e-10)
+        global_min, minimisers = minimise(x->x^2, -10..11, tol=1e-10)
         @test global_min ⊆ 0..1e-20
         @test length(minimisers) == 1
         @test minimisers[1] ⊆ -0.1..0.1
 
         global_min, minimisers = minimise(x->(x^2-2)^2, -10..11)
         @test global_min ⊆ 0..1e-7
         @test length(minimisers) == 2
+        @test sqrt(2) ∈ minimisers[2]
+    end
+
+    @testset "Minimise in 1D using SoretedVector" begin
+        global_min, minimisers = minimise(x->x, -10..10, structure=SortedVector)
+        @test global_min ⊆ -10 .. -9.999
+        @test length(minimisers) == 1
+        @test minimisers[1] ⊆ -10 .. -9.999
+
+        global_min, minimisers = minimise(x->x^2, -10..11, tol=1e-10, structure=SortedVector)
```

Comment:
You can add `for` loop to sets for both `SortedVector` and `HeapedVector`.

Reply:
Thats nice idea.